views/random-beer.hbs

```diff
@@ -0,0 +1,8 @@
+<h1>RANDOM BEER</h1>
+
```

Comment:
Here, I think it would be nicer to have the title of the beer as the title of the page, as opposed to 'random beer'. We already know we chose a random one.